kubernetes/deployment-production.tmpl

```diff
@@ -0,0 +1,225 @@
+              value: production
+            - name: DESIGNATOR_API_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-production-env-vars
+                  key: DESIGNATOR_API_PASSWORD
+            - name: EMAIL_EXPORT_S3_BUCKET
+              value: zooniverse-exports
+            - name: EVENTS_API_USERNAME
+              value: api
+            - name: EVENTS_API_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: EVENTS_API_PASSWORD
+            - name: REDIRECTS_BASE
+              value: https://www.zooniverse.org
+            - name: REDIRECTS_PASSWORD_RESET
+              value: /reset-password
+            - name: REDIRECTS_UNSUBSCRIBE
+              value: /unsubscribe
+            - name: HONEYBADGER_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: HONEYBADGER_API_KEY
+            - name: MAILER_ENABLE_STARTTLS_AUTO
+              value: 'true'
+            - name: MAILER_ADDRESS
+              value: email-smtp.us-east-1.amazonaws.com
+            - name: MAILER_PORT
+              value: 587
+            - name: MAILER_DOMAIN
+              value: zooniverse.org
+            - name: MAILER_AUTHENTICATION
+              value: plain
+            - name: MAILER_USER_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: MAILER_USER_NAME
+            - name: MAILER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: MAILER_PASSWORD
+            - name: NEW_RELIC_APP_NAME
+              value: Panoptes
+            - name: NEW_RELIC_MONITOR_MODE
+              value: 'true'
+            - name: NEW_RELIC_LICENSE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: NEW_RELIC_LICENSE_KEY
+            - name: PAGE_SIZE_LIMIT
+              value: 200
+            - name: PROJECT_REQUEST_BASE_URL
+              value: https://www.zooniverse.org
+            - name: PROJECT_REQUEST_RECIPIENTS
+              value: project-review@zooniverse.org
+            - name: PROJECT_REQUEST_BCC
+              value: i2h6y2f6a7o0x7i9@zooniverse.slack.com
+            - name: SIDEKIQ_HOST
+              value: panoptes-redis.zooniverse.org
+            - name: SIDEKIQ_PORT
+              value: 6379
+            - name: SIDEKIQ_VERBOSE
+              value: false
+            - name: SIDEKIQ_LOGFILE
+              value: ./log/sidekiq.log
+            - name: SIDEKIQ_PIDFILE
+              value: ./tmp/pids/sidekiq.pid
+            - name: SIDEKIQ_CONCURRENCY
+              value: 8
+            - name: SIDEKIQ_TIMEOUT
+              value: 8
```

Comment:
this value should be higher, perhaps 20/30? this is the amount of time before a SIGINT signal is sent to the sidekiq process and it waits before forcefully killing jobs and shutting down.